src/output.ts

```diff
@@ -0,0 +1,96 @@
+import * as path from 'path';
+import { mkdir, stat, writeFile } from 'mz/fs';
+import * as eventToPromise from 'event-to-promise';
+import { spawn as origSpawn, SpawnOptions } from 'child_process';
+import { GeneratedCode, Role } from './types';
+
+const tsconfig = {
+  version: '2.4.2',
+  compilerOptions: {
+    lib: ['es2017', 'esnext'],
+    target: 'es2017',
+    module: 'commonjs',
+    moduleResolution: 'node',
+    emitDecoratorMetadata: true,
+    checkJs: false,
+    allowJs: false,
+    experimentalDecorators: true,
+    downlevelIteration: true,
```

Comment:
No need for `downlevelIteration` with es2017, also `checkJs` and `allowJs` are false by default.

Reply:
OK